docs/api.md

```
@@ -94,67 +172,88 @@ HyperConvergedList contains a list of HyperConverged
 
-## HyperConvergedObsoleteCPUs
+## HyperConvergedSpec
 
-HyperConvergedObsoleteCPUs allows avoiding scheduling of VMs for obsolete CPU models
+HyperConvergedSpec defines the desired state of HyperConverged
 
 | Field | Description | Scheme | Default | Required |
 | ----- | ----------- | ------ | ------- | -------- |
-| minCPUModel | MinCPUModel is not in use Deprecated: This field is not in use and is ignored. | string |  | false |
-| cpuModels | CPUModels is a list of obsolete CPU models. When the node-labeller obtains the list of obsolete CPU models, it eliminates those CPU models and creates labels for valid CPU models. The default values for this field is nil, however, HCO uses opinionated values, and adding values to this list will add them to the opinionated values. | []string |  | false |
+| featureGates | For feature gate details, see [here](#hco-feature-gates) | featuregates.HyperConvergedFeatureGates |  | false |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)